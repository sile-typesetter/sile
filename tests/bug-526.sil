\begin[class=plain,papersize=a6]{document}
\nofolios
\script[src=packages/frametricks]
\script[src=packages/lorem]
\begin[first-content-frame=leftCol]{pagetemplate}
  \frame[id=leftCol,left=left(content),right=50%pw,top=top(content),bottom=bottom(content),next=rightCol]
  \frame[id=rightCol,left=right(leftCol),top=top(content),bottom=bottom(content),width=width(leftCol)]
\end{pagetemplate}
\neverindent
\begin[size=12pt]{font}
% not working using ex values; working using point values
\set[parameter=document.lineskip,value=2em]
% \set[parameter=document.lineskip,value=24pt]
\lorem[words=50,counter=true]\par
\end{font}
\end{document}

