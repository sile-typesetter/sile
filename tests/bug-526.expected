Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	30.0584
Set font 	Gentium Plus;12;400;;normal;;LTR
T	79 82 85 72 80 w=29.2207	(lorem)
Mx 	47.3112
T	20 w=5.6309	(1)
Mx 	56.1506
T	76 83 86 88 80 w=30.2402	(ipsum)
Mx 	89.5994
T	21 w=5.6309	(2)
Mx 	98.4389
T	71 82 79 82 85 w=26.3145	(dolor)
Mx 	127.9619
T	22 w=5.6309	(3)
Mx 	136.8013
T	86 76 87 w=12.0176	(sit)
Mx 	14.8819
My 	64.3709
T	23 w=5.6309	(4)
Mx 	25.5220
T	68 80 72 87 w=24.8203	(amet)
Mx 	55.3516
T	24 w=5.6309	(5)
Mx 	65.9918
T	70 82 81 86 72 87 72 87 88 85 w=52.9922	(consetetur)
Mx 	123.9932
T	25 w=5.6309	(6)
Mx 	134.6334
T	86 68 w=10.1426	(sa)
Mx 	144.7759
T	16 w=4.0430	(-)
Mx 	14.8819
My 	97.6287
T	71 76 83 w=15.8496	(dip)
Mx 	30.7315
T	86 70 76 81 74 w=25.6055	(scing)
Mx 	58.6272
T	26 w=5.6309	(7)
Mx 	66.5483
T	72 79 76 87 85 w=20.9297	(elitr)
Mx 	89.7682
T	27 w=5.6309	(8)
Mx 	97.6893
T	86 72 71 w=16.4180	(sed)
Mx 	116.3975
T	28 w=5.6309	(9)
Mx 	124.3186
T	71 76 68 80 w=24.6387	(diam)
Mx 	14.8819
My 	133.6404
T	20 19 w=11.2617	(10)
Mx 	28.4338
T	81 82 81 88 80 92 w=41.1914	(nonumy)
Mx 	71.9155
T	20 20 w=11.2617	(11)
Mx 	85.4674
T	72 76 85 80 82 71 w=35.4609	(eirmod)
Mx 	123.2186
T	20 21 w=11.2617	(12)
Mx 	136.7706
T	87 72 80 w=19.3125	(tem)
Mx 	156.0831
T	16 w=4.0430	(-)
Mx 	14.8819
My 	169.6522
T	83 82 85 w=17.1445	(por)
Mx 	34.9137
T	20 22 w=11.2617	(13)
Mx 	49.0627
T	76 81 89 76 71 88 81 87 w=42.3926	(invidunt)
Mx 	94.3426
T	20 23 w=11.2617	(14)
Mx 	108.4916
T	88 87 w=10.4883	(ut)
Mx 	121.8672
T	20 24 w=11.2617	(15)
Mx 	136.0162
T	79 68 w=8.7598	(la)
Mx 	144.7759
T	16 w=4.0430	(-)
Mx 	14.8819
My 	205.5467
T	69 82 85 72 w=22.4238	(bore)
Mx 	40.1403
T	20 25 w=11.2617	(16)
Mx 	54.2365
T	72 87 w=9.6738	(et)
Mx 	66.7449
T	20 26 w=11.2617	(17)
Mx 	80.8412
T	71 82 79 82 85 72 w=31.8574	(dolore)
Mx 	115.5332
T	20 27 w=11.2617	(18)
Mx 	129.6294
T	80 68 w=15.1465	(ma)
Mx 	144.7759
T	16 w=4.0430	(-)
Mx 	14.8819
My 	238.8045
T	74 81 68 w=17.9824	(gna)
Mx 	36.1884
T	20 28 w=11.2617	(19)
Mx 	50.7743
T	68 79 76 84 88 92 68 80 w=45.6152	(aliquyam)
Mx 	99.7136
T	21 19 w=11.2617	(20)
Mx 	114.2995
T	72 85 68 87 w=19.9336	(erat)
Mx 	137.5572
T	21 20 w=11.2617	(21)
Mx 	14.8819
My 	274.8162
T	86 72 71 w=16.4180	(sed)
Mx 	34.3357
T	21 21 w=11.2617	(22)
Mx 	48.6333
T	71 76 68 80 w=24.6387	(diam)
Mx 	76.3078
T	21 22 w=11.2617	(23)
Mx 	90.6053
T	89 82 79 88 83 87 88 68 w=43.9160	(voluptua)
Mx 	137.5572
T	21 23 w=11.2617	(24)
Mx 	14.8819
My 	309.1287
T	68 87 w=9.6387	(at)
Mx 	28.9658
T	21 24 w=11.2617	(25)
Mx 	44.6728
T	89 72 85 82 w=22.2480	(vero)
Mx 	71.3661
T	21 25 w=11.2617	(26)
Mx 	87.0730
T	72 82 86 w=16.2129	(eos)
Mx 	107.7311
T	21 26 w=11.2617	(27)
Mx 	123.4381
T	72 87 w=9.6738	(et)
Mx 	137.5572
T	21 27 w=11.2617	(28)
Mx 	14.8819
My 	341.2498
T	68 70 70 88 86 68 80 w=42.1348	(accusam)
Mx 	61.8326
T	21 28 w=11.2617	(29)
Mx 	77.9102
T	72 87 w=9.6738	(et)
Mx 	92.4000
T	22 19 w=11.2617	(30)
Mx 	108.4776
T	77 88 86 87 82 w=24.2637	(justo)
Mx 	137.5572
T	22 20 w=11.2617	(31)
Mx 	14.8819
My 	377.2615
T	71 88 82 w=18.6328	(duo)
Mx 	37.5651
T	22 21 w=11.2617	(32)
Mx 	52.8772
T	71 82 79 82 85 72 86 w=36.4922	(dolores)
Mx 	93.4197
T	22 22 w=11.2617	(33)
Mx 	108.7318
T	72 87 w=9.6738	(et)
Mx 	122.4560
T	22 23 w=11.2617	(34)
Mx 	137.7681
T	72 68 w=11.0508	(ea)
Mx 	148.8189
My 	30.0584
T	22 24 w=11.2617	(35)
Mx 	162.8605
T	85 72 69 88 80 w=32.3848	(rebum)
Mx 	198.0251
T	22 25 w=11.2617	(36)
Mx 	212.0667
T	86 87 72 87 w=18.4395	(stet)
Mx 	233.2860
T	22 26 w=11.2617	(37)
Mx 	247.3276
T	70 79 76 87 68 w=21.3867	(clita)
Mx 	271.4942
T	22 27 w=11.2617	(38)
Mx 	148.8189
My 	63.3162
T	78 68 86 71 w=22.2363	(kasd)
Mx 	173.8333
T	22 28 w=11.2617	(39)
Mx 	187.8730
T	74 88 69 72 85 74 85 72 81 w=51.3691	(gubergren)
Mx 	242.0202
T	23 19 w=11.2617	(40)
Mx 	256.0599
T	81 82 w=12.6563	(no)
Mx 	271.4942
T	23 20 w=11.2617	(41)
Mx 	148.8189
My 	99.3279
T	86 72 68 w=15.6855	(sea)
Mx 	168.2714
T	23 21 w=11.2617	(42)
Mx 	183.3001
T	87 68 78 76 80 68 87 68 w=43.5293	(takimata)
Mx 	230.5964
T	23 22 w=11.2617	(43)
Mx 	245.6250
T	86 68 81 70 87 88 86 w=37.1309	(sanctus)
Mx 	148.8189
My 	132.5858
T	23 23 w=11.2617	(44)
Mx 	162.6007
T	72 86 87 w=14.3086	(est)
Mx 	179.4294
T	23 24 w=11.2617	(45)
Mx 	193.2112
T	79 82 85 72 80 w=29.2207	(lorem)
Mx 	224.9520
T	23 25 w=11.2617	(46)
Mx 	238.7339
T	76 83 86 88 80 w=30.2402	(ipsum)
Mx 	271.4942
T	23 26 w=11.2617	(47)
Mx 	148.8189
My 	168.4803
T	71 82 79 82 85 w=26.3145	(dolor)
Mx 	178.3091
T	23 27 w=11.2617	(48)
Mx 	192.7467
T	86 76 87 w=12.0176	(sit)
Mx 	207.9400
T	23 28 w=11.2617	(49)
Mx 	222.3775
T	68 80 72 87 w=24.8203	(amet)
End page
Finish
